index.html

```diff
@@ -66,8 +66,210 @@ <h1 id="main-header">I’m Alexandra.
                 </ul>
           </div>
         </section>
+    </main>
+
+    <!-- WORK SECTION START  -->
+    <section id="work">
+        <section class="project-section">
+            <figure>
+                <img src="./Images/Snapshoot Portfolio.png" alt="" id="snap-shot-image">
+            </figure>
+            <section class = "project-showing">
+                <h2>Tonic</h2>
+                <ul class="experience">
+                    <li id="canopy-text">CANOPY</li>
+                    <img src="./Images/Counter.png" alt="" class="counter-img">
+                    <li>Back End Dev</li>
+                    <img src="./Images/Counter.png" alt="" class="counter-img">
+                    <li>2015</li>
+                </ul>
```

Comment:
[Optional] In the case of the cards that contain the project information, it could be done as a `div` as they are distinct elements, rather than a list of elements presented on a sequential manner. As my colleague mentioned, `ul` and `li` should be used when you have a _list of elements_ that are related

Reply:
Thank you for all the optional instructions. It is really helpful for me to improve my understanding.